packages/material-ui-envinfo/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "@material-ui/envinfo",
+  "private": false,
```

Comment:
I think this is required to allow `npm publish` instead of `npm publish --access public` .

Reply:
It doesn't seem to make a difference, I was expecting code sandbox to fail on this. It's also to be noted that all packages are published from the `/build` folder.

```json
  "publishConfig": {
    "access": "public"
  },
```

Reply:
There's nothing to build here.

Reply:
I guess we can simply copy the files into the build folder in this case? (same strategy as for the types package)

Reply:
I just saw https://github.com/mui-org/material-ui/pull/23886

Makes sense to have a consistent layout so that we don't have to handle too many edge cases. At least until we have smoke tests for the built packages like https://github.com/mui-org/material-ui/pull/23166

Reply:
Let's fix this in https://github.com/mui-org/material-ui/pull/23886